### PR TITLE
Add link to the feedstock license file

### PR DESCRIPTION
**Pull request by jwillemsen**
Add link to spdx.org for feedstock license

### CODE REVIEW

**Comment by jakirkham** [comment]
As the license file is visible in the top-level directory of the repository, I'm curious why we would want to link to SPDX instead or why such link would be needed at all. Thoughts?

**Comment by jwillemsen** [comment]
If the license is always part of the repo, why not generate a link to the license file?

**Comment by jakirkham** [comment]
Yeah I just assume people can find this easily. That said, no objection to link to it.

**Comment by jakirkham** [comment]
Could you please include a new entry as well?

File: conda_smithy/templates/README.md.tmpl [hunk 1]
```diff
@@ -10,7 +10,7 @@ Home: {{ package.meta.about.home }}
 
 Package license: {{ package.meta.about.license }}
 
-Feedstock license: BSD-3-Clause
+Feedstock license: [BSD-3-Clause](LICENSE.txt)
```
Comment (scopatz) [inline conda_smithy/templates/README.md.tmpl:13]: Doesn't this need to be a full URL?

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:13]: No it's a relative URL based on the feedstock's root directory.

**Review by jakirkham — APPROVED** [review]

File: conda_smithy/templates/README.md.tmpl (hunk 1 above)
Comment (scopatz) [reply conda_smithy/templates/README.md.tmpl:13]: That seems too github specific, to be honest. It wouldn't be helpful if someone opened a file locally

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl:13]: With markdown a relative local link also workels, see https://stackoverflow.com/questions/32563078/how-link-to-any-local-file-with-markdown-syntax/58317580#58317580

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:13]: How about this? Verified to work with [MacDown]( https://macdown.uranusjr.com ) locally.

```suggestion
Feedstock license: [BSD-3-Clause](./LICENSE.txt)
```

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:13]: Actually it works without the `./` locally. So I don't know that it is needed actually.

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl:13]: Ok, so when it works without `./` just let it be as it is

Comment (scopatz) [reply conda_smithy/templates/README.md.tmpl:13]: I'd rather this be a full URL, personally

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:13]: Sounds good. Will wait to see if Anthony has any other thoughts here :)

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl]: Updated the PR to use a full link

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl]: Well that can be tricky if the repo name doesn't match the package name, which does happen when we rename things periodically. Am a little worried about us generating broken links.

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl]: The link generation uses `github.repo_name` which is used more in the README.md, renaming the packaging would keep the same link, only when you change the repo name the links are broken, but that are at that moment all links to github

Comment (scopatz) [reply conda_smithy/templates/README.md.tmpl]: Yeah, this looks good to me

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl]: Could you please test this change on this feedstock ( https://github.com/conda-forge/python-feedstock )?

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl]: Made one fix, attached generated file

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Johnny! 😄

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**